src/warc2zim/content_rewriting/ds.py

```diff
@@ -0,0 +1,269 @@
+## THis file is almost a plain translation from js file
```

Comment:
Many typos / wrong English. Corrected version suggestion:
```
## This file is almost a plain translation from a JavaScript file.
## https://github.com/webrecorder/wabac.js/blob/219830cea0a732bdd72ce100fcfd2f3a1c9c3607/src/rewrite/dsruleset.js
## Partly translated by a human, partly by ChatGPT.

## The ruleset should be kept in sync with the original ones, but there is no easy
## automatic process for that at the moment.
```

Reply:
Fixed (directly in the commit)